modules/programs/navi.nix

```diff
@@ -0,0 +1,85 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+
+  cfg = config.programs.navi;
+
+  yamlFormat = pkgs.formats.yaml { };
+
+  inherit (pkgs.stdenv.hostPlatform) isDarwin;
+
+in {
+  meta.maintainers = [ maintainers.marsam ];
+
+  options.programs.navi = {
+    enable = mkEnableOption "Navi";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.navi;
+      defaultText = literalExpression "pkgs.navi";
+      description = "The package to use for the navi binary.";
+    };
+
+    settings = mkOption {
+      type = yamlFormat.type;
+      default = { };
+      defaultText = literalExpression "{ }";
```

Comment:
No need for this line. The `default` attribute on the line above should be sufficient to produce sensible documentation.
```suggestion
```